### PR TITLE
Add compliant and noncompliant examples of javascript/insecure-cookie@v1.0

### PR DESCRIPTION
**Pull request by swaraj-amazon**
*Description of changes:*
* Rules Added: javascript/insecure-cookie@v1.0
* Added new functions for compliant and non compliant examples

### CODE REVIEW

File: src/javascript/detectors/insecure_cookie/insecure_cookie.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// {fact rule=insecure-cookie@v1.0 defects=1}
+var cookieSession = require('cookie-session')
+var express = require('express')
+var app = express()
+function insecureCookieNoncompliant() {
+    let session =
+    app.use(cookieSession({
```
Comment (ctlai95) [inline src/javascript/detectors/insecure_cookie/insecure_cookie.js:12]: Formatting is a bit strange here, can we remove the new line?

File: src/javascript/detectors/insecure_cookie/insecure_cookie.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+// {/fact}
+
+
+// {fact rule=insecure-cookie@v1.0 defects=0}
+var cookieSession = require('cookie-session')
+var express = require('express')
+var app = express()
+function insecureCookieCompliant() {
+    // Compliant: by default `httpOnly` is set to true and thus makes cookie secure.
+    app.use(cookieSession({
```
Comment (ctlai95) [inline src/javascript/detectors/insecure_cookie/insecure_cookie.js:28]: Can we add `let session =` to keep it consistent?

**Review by ctlai95 — CHANGES_REQUESTED** [review]

File: src/javascript/detectors/insecure_cookie/insecure_cookie.js (hunk 1 above)
Comment (swaraj-amazon) [reply src/javascript/detectors/insecure_cookie/insecure_cookie.js:12]: removed new line

File: src/javascript/detectors/insecure_cookie/insecure_cookie.js (hunk 2 above)
Comment (swaraj-amazon) [reply src/javascript/detectors/insecure_cookie/insecure_cookie.js]: Added session variable

**Review by ctlai95 — APPROVED** [review]

**Pull request merged**